### PR TITLE
8.0

### PR DESCRIPTION
**Pull request by EvgeniyPatlan**

### CODE REVIEW

**Review by hors — APPROVED** [review]

File: local/build-binary [hunk 1]
```diff
@@ -150,7 +150,7 @@ pushd ${WORKDIR}
         -DWITH_PROTOBUF=bundled \
         -DWITH_RAPIDJSON=bundled \
         -DWITH_ICU=bundled \
-	-DWITH_EDITLINE=system \
+	      -DWITH_EDITLINE=bundled \
```
Comment (hors) [inline local/build-binary:153]: please remove spaces

**Review by delgod — APPROVED** [review]

**Comment by laurynas-biveinis** [comment]
Why this and not -DWITH_READLINE=system?

**Comment by hors** [comment]
Evgeniy, it does not work with '-DWITH_READLINE=system' 
https://ps3.cd.percona.com/job/fb-mysql-server-8.0-pipeline/9/console

Maybe not all system libs are installed, we need to check it.

**Comment by EvgeniyPatlan** [comment]
@hors please create ticket and check dependencies. I assume  readline package is not installed

**Pull request merged**